.github/workflows/codeql.yml

```
@@ -13,7 +13,7 @@ name: "CodeQL Advanced"
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "dev" ]
```

yamllint

[error] too many spaces inside brackets

(brackets)